x-pack/plugins/metrics_entities/.i18nrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "prefix": "metricsEntities",
+  "paths": {
+    "metricsEntities": "."
+  },
+  "translations": ["translations/ja-JP.json"]
```

Comment:
Is this missing `translations/zh-CN.json`? This is an artifact of the plugin generator right? Not seeing many other plugins use this pattern, looks like they're added here?

https://github.com/elastic/kibana/blob/eddf1c94b1c76492e619bef7557a04e8cbbea7d2/.i18nrc.json

Reply:
I think so as well. I am going to update to this pattern instead.